natural-language-understanding/v1.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+'use strict';
+
+var requestFactory = require('../lib/requestwrapper');
+var pick = require('object.pick');
+var util = require('util');
+var BaseService = require('../lib/base_service');
+
+/**
+ * NaturalLanguageUnderstanding
+ * @param options
+ * @constructor
+ */
+function NaturalLanguageUnderstandingV1(options) {
+  BaseService.call(this, options);
+}
+util.inherits(NaturalLanguageUnderstandingV1, BaseService);
+NaturalLanguageUnderstandingV1.prototype.name = 'natural_language_understanding';
+NaturalLanguageUnderstandingV1.prototype.version = 'v1';
+NaturalLanguageUnderstandingV1.URL = 'https://gateway-s.watsonplatform.net/natural-language-understanding/api';
```

Comment:
the `.URL` var is special and is used by the base service.  If I change it to NATURAL_LANGUAGE_UNDERSTANDING_URL I'll have to modify the base service.